rebar.config

```diff
@@ -1,33 +1,29 @@
 %%-*- mode: erlang -*-
-{sub_dirs, ["rel"]}.
-
 {cover_enabled, true}.
 
 {erl_opts, [debug_info, fail_on_warning,
             {platform_define, "^[0-9]+", namespaced_types},
-            {i, "./deps/dns/include"},
-            {i, "../dns/include"},
             {parse_transform, lager_transform}]}.
 
-{deps, [
-        {parse_trans, ".*", {git, "git://github.com/uwiger/parse_trans.git", {branch, "master"}}},
-        {lager, ".*", {git, "git://github.com/basho/lager.git", {branch, "master"}}},
-        {recon, ".*", {git, "git://github.com/ferd/recon.git", {branch, "master"}}},
-        {folsom, ".*", {git, "git://github.com/folsom-project/folsom.git", {branch, "master"}}},
-        {poolboy, ".*", {git, "git://github.com/devinus/poolboy.git", {branch, "master"}}},
-        {jsx, ".*", {git, "git://github.com/talentdeficit/jsx.git", {branch, "master"}}},
+{deps, [lager,
+        recon,
+        folsom,
+        jsx,
         {dns, ".*", {git, "git://github.com/dnsimple/dns_erlang.git", {branch, "master"}}},
```

Comment:
Does it make sense to try to get dns_erlang into hex as well?

Reply:
Yup, I have a PR related to that open as well.

Reply:
This may be of interest, all this started because we are using erldns in the example project (https://github.com/adoptingerlang/service_discovery) for https://adoptingerlang.org/